### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by muhenge**
# Work done

1. I created a function to get data from [OpenWeather](https://openweathermap.org/) API, and I used the same function to render needed data into HTML as well as performing Celsius to Fahrenheit  calculation, based on the user input
2. I created another function to call the first one(get data), with a click event listener to processes the function on click.
3. I exported the function in index.js.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    "transfer-webpack-plugin": "^0.1.4",
+    "url-loader": "^4.1.1",
+    "webpack": "^5.24.2",
+    "webpack-cli": "^4.5.0",
+    "webpack-dev-server": "^3.11.2"
+  },
+  "scripts": {
+    "build": "webpack",
+    "start:dev": "webpack serve"
+  },
```
Comment (addod19) [inline package.json]: Let us kindly improve our script, so we can run commands in both development and production mode
```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @muhenge, congratulations on this project :fireworks: The app returns data to the browser :+1: The README is great :slightly_smiling_face: However, before approving this project, we need to work on the following comments:
- `PR` - This is one of the important elements of having a professional repo, the PR must include a title and a brief description of the project. Currently, we have nothing, let us kindly fix it
- `Application` - The application is missing the following requirements
  - Toggle functionality, to toggle between the Celsius and Fahrenheit
  - Changes appearance based on received data

[Otional]
- Kindly implement the loading functionality so we get some effect while waiting for the data
- Kindly implement the functionality where the user's location is used to retrieve data when the app starts

Once these are resolved, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

File: package.json (hunk 1 above)
Comment (muhenge) [reply package.json]: Thanks for the feedback.

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Impressive work. 💻 👀 ✨ 

- Excellent job adding the necessary features requested in the previous code review.

## However, there are some changes required 👀 🧰 🔨 🔧 

- As mentioned earlier, please add a proper title and description to this Pull Request.

### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Review by arslanbisharat — APPROVED** [review]
# Approved 🥇 

Thank you for submitting this project. You have done a great job!

- [x] Your commit messages are very descriptive too. I am very happy to see that.  👏 
- [x] No Linter error found. 👏 
- [x] Met Minimum requirements to pass the project.  

I am very happy to review your project. Looking forward to amazing new projects from your side. 

**Best,
Arslan Bisharat**
_You can always contact me on slack at @arslanbisharat 📞_

**Pull request merged**